src/ai/susi/server/api/aaa/ResendVerificationLinkService.java

```diff
@@ -32,12 +32,13 @@
 /**
  * Created by dravit on 25/7/17.
  * parameter : emailId
- * sample request : 
+ * sample request :
  * http://127.0.0.1:4000/aaa/resendVerificationLink.json?emailId=dravit.lochan@gmail.com
```

Comment:
@mariobehling This is just a sample request. In my earlier PR, I added example@example.com as  a sample API call but had to change it. Please direct me what email id should I put in sample requests.